CHANGELOG.rst

```diff
@@ -2,6 +2,30 @@
 Changelog
 *********
 
+4.0.0 -- 2024-10-28
+===================
+
+Features
+--------
+* Add support for constructs from the `AWS Cryptographic Material Providers Library (MPL) <https://github.com/aws/aws-cryptographic-material-providers-library>`_.
+  The MPL contains new constructs for encrypting and decrypting your data.
+  We highly recommend installing the MPL. See `Installing <https://github.com/aws/aws-encryption-sdk-python/tree/master?tab=readme-ov-file#installation>`_ for instructions.
+
+Breaking Changes
+^^^^^^^^^^^^^^^^
+* Messages constructed with the MPL's Required Encryption Context Cryptographic Materials Manager ("required EC CMM")
+  will not be readable from versions of the ESDK <4.0.0.
+  The MPL introduces the "required EC CMM" as a new construct for protecting your data.
+  It requires that, for a specified set of encryption context keys,
+  a decryptor must supply the same encryption context pairs that were used when encrypting the message.
```

Comment:
Would it be better to say what is does?

Like

```suggestion
The required EC CMM will not store the configured encryption context in the ESDK message,
this will require that decryptors must supply
these same encryption context pairs that were used when decrypting the message.
```